met-web/src/components/userManagement/userDetails/AddToEngagement.tsx

```diff
@@ -36,12 +36,12 @@ import { HTTP_STATUS_CODES } from 'constants/httpResponseCodes';
 export const AddToEngagementModal = () => {
     const { savedUser, addUserModalOpen, setAddUserModalOpen, getUserMemberships, getUserDetails } =
         useContext(UserDetailsContext);
-    const userHasGroup = savedUser?.groups && savedUser?.groups.length > 0;
+    const userHasGroup = savedUser?.composite_roles && savedUser?.composite_roles.length > 0;
```

Comment:
Very minor obviously but you could switch this to camelCase when working in JS

```suggestion
    const userHasGroup = savedUser?.compositeRoles && savedUser?.compositeRoles.length > 0;
```

Reply:
Leaving this as is because this is a existing property name as provided by the backend